go.mod

```diff
@@ -21,22 +21,24 @@ require (
 	github.com/containers/storage v0.0.0-20190726081758-912de200380a // indirect
 	github.com/davecgh/go-spew v1.1.1
 	github.com/docker/distribution v2.7.1+incompatible
-	github.com/docker/docker v17.12.0-ce-rc1.0.20200916142827-bd33bbf0497b+incompatible
+	github.com/docker/docker v20.10.3+incompatible
```

Comment:
We have a `replace` for docker/docker dependency that can be now dropped too, most likely. Wanna do it in this PR of followup?

Reply:
Will do it here.